src/bundle/Resources/public/js/scripts/sidebar/btn/content.edit.js

```diff
@@ -0,0 +1,13 @@
+(function () {
+    const editButton = document.querySelector('.ez-btn--edit');
+    const languageRadioOption = document.querySelector('.ez-extra-actions--edit.ez-extra-actions--prevent-show [type="radio"]');
+
+    if (null === languageRadioOption) {
```

Comment:
You can do just `if (!languageRadioOption) {`

Reply:
Please, no yoda comparisons in JS.